src/NzbDrone.Core.Test/DecisionEngineTests/UpgradeDiskSpecificationFixture.cs

```diff
@@ -30,35 +28,35 @@ public class UpgradeDiskSpecificationFixture : CoreTest<UpgradeDiskSpecification
         public void Setup()
         {
             Mocker.Resolve<QualityUpgradableSpecification>();
-            _upgradeDisk = Mocker.Resolve<UpgradeDiskSpecification>();
 
-            _firstFile = new TrackFile { Quality = new QualityModel(Quality.MP3_512, new Revision(version: 2)), DateAdded = DateTime.Now };
-            _secondFile = new TrackFile { Quality = new QualityModel(Quality.MP3_512, new Revision(version: 2)), DateAdded = DateTime.Now };
+            _firstFile = new TrackFile { Quality = new QualityModel(Quality.FLAC, new Revision(version: 2)), DateAdded = DateTime.Now };
+            _secondFile = new TrackFile { Quality = new QualityModel(Quality.FLAC, new Revision(version: 2)), DateAdded = DateTime.Now };
 
-            //var singleEpisodeList = new List<Album> { new Album { EpisodeFile = _firstFile, EpisodeFileId = 1 }, new Album { EpisodeFile = null } };
-            //var doubleEpisodeList = new List<Album> { new Album { EpisodeFile = _firstFile, EpisodeFileId = 1 }, new Album { EpisodeFile = _secondFile, EpisodeFileId = 1 }, new Episode { EpisodeFile = null } };
-
-            var singleEpisodeList = new List<Album> { new Album { }, new Album {} };
+            var singleEpisodeList = new List<Album> { new Album {}};
             var doubleEpisodeList = new List<Album> { new Album {}, new Album {}, new Album {} };
 
-
-            var fakeSeries = Builder<Artist>.CreateNew()
+            var fakeArtist = Builder<Artist>.CreateNew()
                          .With(c => c.Profile = new Profile { Cutoff = Quality.MP3_512, Items = Qualities.QualityFixture.GetDefaultQualities() })
                          .Build();
 
+            Mocker.GetMock<IMediaFileService>()
+                  .Setup(c => c.GetFilesByAlbum(It.IsAny<int>(), It.IsAny<int>()))
+                  .Returns(new List<TrackFile> { _firstFile, _secondFile });
+
             _parseResultMulti = new RemoteAlbum
             {
-                Artist = fakeSeries,
-                ParsedAlbumInfo = new ParsedAlbumInfo { Quality = new QualityModel(Quality.MP3_192, new Revision(version: 2)) },
+                Artist = fakeArtist,
+                ParsedAlbumInfo = new ParsedAlbumInfo { Quality = new QualityModel(Quality.MP3_256, new Revision(version: 2)) },
                 Albums = doubleEpisodeList
             };
 
             _parseResultSingle = new RemoteAlbum
             {
-                Artist = fakeSeries,
-                ParsedAlbumInfo = new ParsedAlbumInfo { Quality = new QualityModel(Quality.MP3_192, new Revision(version: 2)) },
+                Artist = fakeArtist,
+                ParsedAlbumInfo = new ParsedAlbumInfo { Quality = new QualityModel(Quality.MP3_256, new Revision(version: 2)) },
```

Comment:
I noticed several of these changes between music qualities. They don't affect testing results or anything, right?